spec/index.html

```diff
@@ -1287,57 +1287,51 @@ <h2>Security Considerations</h2>
   <p>The RDF Abstract Syntax is not used directly for conveying information,
     although concrete serialization forms are specifically intended to do so.</p>
 
-  <p>Applications MAY evaluate given data to infer more assertions or to dereference IRIs,
+  <p>Applications MAY evaluate given data to infer more assertions or to dereference <a>IRIs</a>,
     invoking the security considerations of the scheme for that IRI.
     Note in particular, the privacy issues in [[RFC3023]] section 10 for HTTP IRIs.
     Data obtained from an inaccurate or malicious data source may lead to inaccurate or misleading conclusions,
     as well as the dereferencing of unintended IRIs.
-    Care must be taken to align the trust in consulted resources with the sensitivity of the intended use of the data;
-    inferences of potential medical treatments would likely require
-    different trust than inferences for trip planning.</p>
+    Care must be taken to align the trust in consulted resources with the sensitivity of
+    the intended use of the data;
+    inferences of potential medical treatments would likely require different trust than inferences
+    for trip planning.</p>
 
   <p>RDF is used to express arbitrary application data;
     security considerations will vary by domain of use.
     Security tools and protocols applicable to text
-    (e.g., PGP encryption, MD5 sum validation, password-protected compression)
+    (for example, PGP encryption, checksum validation, password-protected compression)
     may also be used on RDF documents.
     Security/privacy protocols must be imposed which reflect the sensitivity of the embedded information.</p>
 
-  <p>RDF can express data which is presented to the user, for example, RDF Schema labels.
-    Applications rendering strings retrieved from untrusted RDF documents must ensure
-    that malignant strings may not be used to mislead the reader.
-    The security considerations in the media type registration for
-    XML ([[RFC3023]] section 10) provide additional guidance around the
-    expression of arbitrary data and markup.</p>
+  <p>RDF can express data which is presented to the user, such as RDF Schema labels.
+    Applications rendering strings retrieved from untrusted RDF documents,
+    or using unescaped characters,
+    SHOULD ensure that malignant strings may not be used to mislead the reader.
```

Comment:
```suggestion
    SHOULD ensure that strings may not be used to mislead the reader.
```
(I thought I'd suggested this before but that seems t have got lost)

Reply:
How is any app supposed to review all strings and ensure that they are not misleading, any more than malignant?

As I've said in at least one thread on at least one repo (but I think it's several at this point), warnings and other mechanisms _may_ be appropriate, but it's unclear to me whether there's a good place for any of these to be displayed.

Reply:
The suggestion is to remove the word. Nothing more. it is unnecessary and adds nothing because the text after explains the intent to mislead.

Reply:
Yes, I understand your suggestion. My point is that the advisory remains impossible without "malignant", so I don't see the point of removing that word without changing the rest of the sentence.

Reply:
We have parallel text here at in https://github.com/w3c/rdf-n-quads/pull/17. @TallTed suggested changes there, which I've applied. If we can focus on the N-Quads text until fully approved, we can synchronized those changes back here, and ultimately to the other affected specifications.

Reply:
The relevant text is (currently) at insert lines 329-332 of PR #17.